EOSBasic/Plugins/UEOS/Source/UEOS/Private/Authentication/Authentication.cpp

```diff
@@ -24,13 +23,6 @@ void UEOSAuthentication::Login( ELoginMode LoginMode, FString UserId, FString Us
 	{
 		MessageText = FString::Printf( TEXT( "[EOS SDK | Plugin] Can't Log In - EOS SDK Not Initialized!." ) );
 		UE_LOG( UEOSLog, Warning, TEXT( "%s" ), *MessageText );
-
-		// Broadcast the Login Failure Delegate.
-		if( UEOSManager::GetAuthentication()->OnUserLoginFail.IsBound() )
-		{
-			UEOSManager::GetAuthentication()->OnUserLoginFail.Broadcast();
```

Comment:
Need to have these back in, otherwise BPs can't get notified of Login Fails.

Reply:
It is there if you view the whole file - Git was just acting funky